src/main/java/spotify/api/enums/RepeatType.java

```diff
@@ -1,12 +1,27 @@
 package spotify.api.enums;
 
+/**
+ * This class contains all suitable repeat types.
+ * <p>
+ * For further information see: <a href="https://developer.spotify.com/documentation/web-api/reference/player/set-repeat-mode-on-users-playback/">
+ * Set Repeat Mode On User's Playback</a>.
+ *
+ * @author Jiankai Zheng (jk.zheng@hotmail.com)
+ * @since 1.0.0
```

Comment:
Not added in 1.0.0

Reply:
You say not added in 1.0.0, but I show it was added in 1.0.0? Unless I am reading the releases wrong? I went to Comparing1.0.0...main and saw it there

Reply:
See my comment https://github.com/jzheng2017/spotify-web-api-wrapper/pull/12#discussion_r558598820